handbook/README.md

```diff
@@ -203,6 +202,12 @@ The Fleet handbook is the central guide for how we run the company. As part of o
 
 [Slack channels](./customers.md#slack-channels)
 
+### :handshake: Sales
```

Comment:
You can copy and paste the emoji from here: https://emojipedia.org/handshake/

(FYI the `:foo:` thing with colons only works in Slack and in actual GitHub comments/issues -- not in the code.)

Reply:
Thank you